src/lib.rs

```diff
@@ -219,7 +218,8 @@ pub trait Events {
 
 impl Cluster {
     pub fn new() -> Cluster {
-        Cluster { host: Url::parse(DEFAULT_HOST).unwrap() }
+        let kubernetes_api_host: &str = &env::var("KUBERNETES_API_HOST").expect("KUBERNETES_API_HOST not set.");
```

Comment:
Can we keep the default as a fallback? This was meant to work with kubectl proxy which listens on localhost 8001. I'd prefer a sensible default than a runtime error.